docs/GOVERNANCE.md

```diff
@@ -139,18 +149,14 @@ If a vote is taken, the follow rules will be followed:
 
 The specifications produced will adhere to the following:
 
-- The versioning scheme used will follow [semver](https://semver.org/)
-- All normative specifications, and the Primer, will be grouped together into a
-  single logical unit and released at the same time, at the same version number.
-  This is true regardless of whether each individual document actually changed
-  during the release cycle.
-- When a new release of a specification is ready, it will be given a version
-  number matching the appropriate semver version string but with a suffix of
-  `-rc#` (release candidate). This will indicate that the authors believe it
-  is ready for final release but it needs to go through a testing period to
-  allow for broader testing before it promoted to its final version number.
-  This will be true for updates to existing specifications and for new
-  specifications.
+- The versioning scheme used will follow [semver](https://semver.org/).
```

Comment:
Not blocking for this PR, but we could probably change all of this into the present tense now.